website_sale_address_name_description/views/website_sale_address_view.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="address" inherit_id="website_sale.address">
+        <xpath expr="//label[@for='name']" position="after">
+            <span>(Name, Company Name)</span>
+        </xpath>
+        <xpath expr="//input[@name='name']" position="after">
+            <t t-out="request.env.company.address_name_text" />
```

Comment:
Can't we use odoo standard address fields instead of adding new field?

Reply:
Thank you for your review.  
This field is not an address, but text to add a note by the address.

Add a note like this.
"If you enter the name of yor company, be sure to enter the name of the person in charge as well."